<ui version="4.0" >
 <class>PrefInterface</class>
 <widget class="QWidget" name="PrefInterface" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>&amp;Interface</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="4" column="0" colspan="2" >
        <widget class="Line" name="line" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2" >
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Instances</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="single_instance_check" >
            <property name="text" >
             <string>&amp;Use only one running instance of SMPlayer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="port_label" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>SMPlayer will listen to this &amp;port to receive commands from other instances:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap" >
             <bool>true</bool>
            </property>
            <property name="buddy" >
             <cstring>server_port_spin</cstring>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>61</width>
                <height>31</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSpinBox" name="server_port_spin" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="maximum" >
               <number>10000</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>270</width>
                <height>31</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" colspan="2" >
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="2" column="2" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="style_label" >
           <property name="text" >
            <string>St&amp;yle:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
           <property name="buddy" >
            <cstring>style_combo</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="2" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="textLabel1_18" >
           <property name="text" >
            <string>L&amp;anguage:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
           <property name="buddy" >
            <cstring>language_combo</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="textLabel1_19" >
           <property name="text" >
            <string>Ico&amp;n set:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
           <property name="buddy" >
            <cstring>iconset_combo</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="2" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1" >
          <widget class="MyComboBox" name="language_combo" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="MyComboBox" name="iconset_combo" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="MyComboBox" name="style_combo" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item rowspan="2" row="0" column="0" >
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Seeking</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="SeekWidget" native="1" name="seek1" />
            </item>
            <item>
             <widget class="SeekWidget" native="1" name="seek2" />
            </item>
            <item>
             <widget class="SeekWidget" native="1" name="seek3" />
            </item>
            <item>
             <widget class="SeekWidget" native="1" name="seek4" />
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Recent files</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1_15" >
            <property name="text" >
             <string>Ma&amp;x. items</string>
            </property>
            <property name="wordWrap" >
             <bool>false</bool>
            </property>
            <property name="buddy" >
             <cstring>recents_max_items_spin</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="recents_max_items_spin" >
            <property name="maximum" >
             <number>20</number>
            </property>
            <property name="minimum" >
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Main window</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Auto&amp;resize:</string>
            </property>
            <property name="buddy" >
             <cstring>mainwindow_resize_combo</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="mainwindow_resize_combo" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text" >
              <string>Never</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Whenever it's needed</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Only after loading a new video</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="save_size_check" >
            <property name="text" >
             <string>R&amp;emember position and size</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0" colspan="2" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Default font:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="default_font_edit" >
           <property name="readOnly" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="changeFontButton" >
           <property name="text" >
            <string>&amp;Change...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0" colspan="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
  <customwidget>
   <class>SeekWidget</class>
   <extends>QWidget</extends>
   <header>seekwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>single_instance_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>port_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>116</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel" >
     <x>126</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>single_instance_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>server_port_spin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>55</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel" >
     <x>98</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
